docs/sources/reference/components/prometheus.exporter.snowflake.md

```diff
@@ -20,18 +22,34 @@ prometheus.exporter.snowflake "LABEL" {
 }
 ```
 
+### RSA Authentication
+
+```alloy
+prometheus.exporter.snowflake "LABEL" {
+    account_name =         ACCOUNT_NAME
+    username =             USERNAME
+    private_key_path =     /PATH/TO/rsa_key.p8
+    private_key_password = PASSWORD
+    warehouse =            WAREHOUSE
```

Comment:
```suggestion
    account_name =         <ACCOUNT_NAME>
    username =             <USERNAME>
    private_key_path =     <PATH_TO_rsa_key.p8>
    private_key_password = <PASSWORD>
    warehouse =            <WAREHOUSE>
```

For consistency with the direction the docs are taking, we should use the placeholder variable notation defined here: https://grafana.com/docs/writers-toolkit/write/style-guide/write-for-developers/#placeholder-variables

Example with this style implemented: https://grafana.com/docs/alloy/latest/reference/components/discovery.azure/#example

This is still a work in progress... and many topics still need to be update.